CartographyTests/OperatorTests.swift

```diff
@@ -44,7 +44,7 @@ class OperatorTests: XCTestCase {
         var constraint: NSLayoutConstraint!
 
         layout(view) { view in
-            constraint = view.width == (view.superview!.width + 100) * 2 + 5
+            constraint = view.width == (view.superview!.width + 100) * 2 + 5; return
```

Comment:
Unfortunately, these give me an _expression too complex_ error without the `return` statement.